addons/web/static/src/js/libs/popper.js

```diff
@@ -0,0 +1,5 @@
+odoo.define('/web/static/lib/bootstrap/js/popover.js', function () {
+    "use strict";
+    //The pop-up will be overflowed on the right and bottom.
+    Popper.Defaults.modifiers.preventOverflow.priority = ['right', 'left', 'bottom', 'top'];
+});
```

Comment:
- Can you use the new JS module system as GED described in its mail a while ago? https://www.odoo.com/documentation/master/reference/javascript_reference.html#native-javascript-module
- Remove the comment
- Do not indent the first level of JS modules